src/units/execution.js

```diff
@@ -82,6 +84,10 @@ export class ExecutionUnit extends mix(BaseUnit).with(HashedInputArrayMixin) {
         this.render(this.context, true);
     }
 
+    setInput(input) {
```

Comment:
What happened her?

Reply:
setInput was missing, forcing the new unit test to fail. Seems it was never needed before, so had never been created.

Reply:
was not necessary in web-app, but needed for wode.js test